contrib/freshmaker-cli

```diff
@@ -225,6 +249,55 @@ def get_freshmaker_events_by_ids(api_url, ids):
     return events
 
 
+def get_advisories_from_errata(errata_url, days=7):
+    """
+    Sends the GET request to the preset Errata filter. Filter shows RHSA in SHIPPED_LIVE state
+    ordered by date of shipping (from the latest to the oldest). Returns the list of advisories
+    that are IMPORTANT or CRITICAL and the date of shipping is older given number of days before the
+    current date.
+    """
+    if days > 7:
+        print("The interval between dates should not exceed 7 days")
+        sys.exit(1)
+    page = 1
+    krb_auth = HTTPKerberosAuth()
+    advisories = []
+    while True:
+        print("Getting advisories from  Errata page", page)
+        r = requests.get(errata_url, auth=krb_auth, timeout=30, params={page: page})
+        r.raise_for_status()
+        data = r.json()
+        date_from = date.today() - timedelta(days)
```

Comment:
Could you move this outside of the loop? Sorry for missing this earlier.

Reply:
Fixed :)